.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+<!-- Provide a general summary of your changes in the title above
+
+Please read our contribution guidelines and sign the Contributor License
+Agreement (CLA) before submitting the pull request. Also, check if there are no
+other open pull requests targeting the same issue. -->
+
+#### Description
+<!-- Describe your changes in detail. -->
+
+##### Category
+<!-- Tell us what type of issue does your pull request target.
+You can uncomment one of the following options: -->
+
+<!-- Bug Fix<br> -->
+<!-- New Feature<br> -->
+<!-- Improvement<br> -->
```

Comment:
Add "Refactor". (And as discussed in MM, I think it is enough to only keep one of "New Feature" and "Improvement".)

Reply:
- New Feature and Improvement and merged into a single template (called improvement).
- Instead of `Refactor`, added `Implementation Task`, since this is the newly added issue type.
See this commit -  1c2dcf9403c97c826b7fa701b7398c9a3a379363 for changes.